src/utils/element-utils.ts

```diff
@@ -1,8 +1,8 @@
-import { TOOLBAR_ID } from '../constants'
+import { TOOLBAR_CONTAINER_CLASS, TOOLBAR_ID } from '../constants'
 
 export function isElementInToolbar(el: Element): boolean {
     // NOTE: .closest is not supported in IE11 hence the operator check
-    return el.id === TOOLBAR_ID || !!el.closest?.('#' + TOOLBAR_ID)
```

Comment:
this was only working previously by accident because there is also `ph-no-capture` which autocapture obeys

`closest` does not escape shadow roots and TOOLBAR_ID is outside the shadow root.

switched to a selector inside the shadow root (the selector provided is at the root of the toolbar inside the shadow root

e.g.

```
<div id="TOOLBAR_ID">
    <shadow_root>
       <div class=TOOLBAR_CONTAINER_CLASS>
```